controllers/articles.controllers.js

```diff
@@ -0,0 +1,14 @@
+const { selectArticleById } = require("../models/articles.models");
+
+function getArticleById(req, res, next) {
+  const articleId = req.params.article_id;
+  selectArticleById(articleId)
+    .then((articleData) => {
+      res.status(200).send({ articleData });
```

Comment:
I would probably just call it `article` instead of `articleData` but up to you 🙂